base/ca/src/main/java/org/dogtagpki/legacy/server/policy/extensions/NSCertTypeExt.java

```diff
@@ -243,12 +243,13 @@ public PolicyResult applyCert(Request req, X509CertInfo certInfo) {
                 if (bits[j])
                     break;
             if (bits == null || j == bits.length) {
-                if (!mSetDefaultBits) {
+                if (mSetDefaultBits)
+                    bits = DEF_BITS;
+                else {
                     logger.debug(
                             "NSCertTypeExt: no bits requested, not setting default.");
                     return PolicyResult.ACCEPTED;
-                } else
-                    bits = DEF_BITS;
+                }
```

Comment:
Should we keep the if-block since it returns and unindent the else-block instead?